src/temporal-types.js

```diff
@@ -142,11 +142,11 @@ export class LocalTime {
   }
 
   /**
-   * Create a local time object from the given standard JavaScript `Date` and optional nanoseconds.
+   * Create a `LocalTime` object from the given standard JavaScript `Date` and optional nanoseconds.
```

Comment:
You can use class reference `{@link DateTime}` instead. Same for the others changes.